pkg/identity/key/global_identity.go

```diff
@@ -70,3 +72,9 @@ func (gi *GlobalIdentity) PutValue(key, value any) allocator.AllocatorKey {
 func (gi *GlobalIdentity) Value(key any) any {
 	return gi.metadata[key]
 }
+
+func GetCIDKeyFromLabels(allLabels map[string]string, source string) *GlobalIdentity {
```

Comment:
Perhaps we should add some tests on this convenience call?

Reply:
hey @derailed, thank you for your review. I've added a few tests.